### ISSUE TITLE
Can't read saves after transferring files

### ISSUE THREAD

**Issue by shannonobrien** [DarwinBaker/AATool#3]
Thanks very much for your tool, I've been using it to track my advancements while playing Minecraft. Unfortunately it looks like I hit an issue that causes it to be unable to read the advancements.

Here's what I did:

1. Play Minecraft on a Windows PC
2. Transfer my world to a Mac
3. Play on the Mac
4. Transfer the files back
5. Play Minecraft on the Windows PC

But after transferring the files, the AATool is no longer able to read the advancements info. It just shows "Not Currently Reading a Save", or it might show "Reading <name of world>" but it doesn't show the advancements.

The strange thing is that the advancements JSON file is definitely there, and it looks valid:

![Screenshot 2021-06-05 162817](https://user-images.githubusercontent.com/23616602/120907973-0f6ab480-c61b-11eb-9a9d-e71f77f1c9b0.png)

Is there some way I can see a log of what file it's trying to read, or any errors it's running into? Or could I manually tell it which JSON file to read?

Thank you for the time and effort you put into this tool!

**Comment by DarwinBaker** [comment]
Thank you so much for the support, it means everything! That's very odd indeed, I'm not sure what might be causing that. My only thought is maybe something to do with the differences in the handling of invisible newlines/carriage returns on mac vs windows? Although the way I read the JSON file _shouldn't_ care. 

Unfortunately I don't own a mac for testing, so if you get the chance, could you possibly attach the problematic advancements JSON so I can do some testing? Also it couldn't hurt to double check the save path settings or reset stuff to default, I can't even begin to tell you how many times I've thought my path was set to something it actually wasn't. Best of luck, I'll be looking into it!

**Comment by shannonobrien** [comment]
Thanks for the response. I did try to change the save path settings and reset everything to default, but it still couldn't read the save file.

Here is the advancements JSON file in case it helps:

[be442736-343d-483e-8bf1-22e455fb1224.zip](https://github.com/DarwinBaker/AATool/files/6604549/be442736-343d-483e-8bf1-22e455fb1224.zip)

**Comment by DarwinBaker** [comment]
The issue gets even more mysterious. Mine has no problem reading the file, so I'm not really sure what might be going on. Maybe try re-naming the world folder to update the modify date? And is your .minecraft folder in the AppData folder of the user account you're logged into when running the tool? (I would assume so). And I take it your advancements show up just fine in-game?

I attached the ones I get just to make sure they're correct:
![tracker](https://user-images.githubusercontent.com/51175243/120941523-2a0f5d00-c6f1-11eb-93df-9b847534b7f3.png)

**Comment by shannonobrien** [comment]
I tried renaming the world folder, but nothing changed. I also checked in-game, and my advancements are there. I also confirmed that I'm running v1.2.4.1 (I downloaded the latest from the Releases page).

It's odd, because I can create other new worlds on this same computer, and the app is able to read the advancements for those just fine. So I don't think it's a problem with the AppData folder location. If there's any more info I can provide, let me know.
